src/storage/copier/rel_copier.cpp

```diff
@@ -392,70 +392,70 @@ void RelCopier::putPropsOfLineIntoColumns(RelCopier* copier,
         auto stringToken =
             currentToken->get()->ToString().substr(0, BufferPoolConstants::PAGE_4KB_SIZE);
         const char* data = stringToken.c_str();
-        switch (properties[propertyIdx].dataType.typeID) {
+        switch (properties[propertyID].dataType.typeID) {
         case INT64: {
             auto val = TypeUtils::convertStringToNumber<int64_t>(data);
-            putValueIntoColumns(propertyIdx, directionTablePropertyColumns, nodeIDs,
+            putValueIntoColumns(propertyID, directionTablePropertyColumns, nodeIDs,
                 reinterpret_cast<uint8_t*>(&val));
         } break;
         case INT32: {
             auto val = TypeUtils::convertStringToNumber<int32_t>(data);
-            putValueIntoColumns(propertyIdx, directionTablePropertyColumns, nodeIDs,
+            putValueIntoColumns(propertyID, directionTablePropertyColumns, nodeIDs,
                 reinterpret_cast<uint8_t*>(&val));
         } break;
         case INT16: {
             auto val = TypeUtils::convertStringToNumber<int16_t>(data);
-            putValueIntoColumns(propertyIdx, directionTablePropertyColumns, nodeIDs,
+            putValueIntoColumns(propertyID, directionTablePropertyColumns, nodeIDs,
                 reinterpret_cast<uint8_t*>(&val));
         } break;
         case DOUBLE: {
             auto val = TypeUtils::convertStringToNumber<double_t>(data);
-            putValueIntoColumns(propertyIdx, directionTablePropertyColumns, nodeIDs,
+            putValueIntoColumns(propertyID, directionTablePropertyColumns, nodeIDs,
```

Comment:
hmmm should we open an issue about revisit test coverage at some point? Seems we don't have test for DOUBLE rel property and BOOL rel property

Reply:
i think it is ok not to test double/bool rel property. If we test each datatype on node column, rel column, rel lists, our testing code will explode.